src/views/Samples/Page.tsx

```diff
@@ -1,21 +1,24 @@
 import topArrow from "images/arrow-down.svg";
 import bottomArrow from "images/arrow-up.svg";
-import React from "react";
+import React, { useState } from "react";
 import { Link } from "react-router-dom";
 import PageHeader from "../../components/Header/PageHeader";
-import { FilterCollection } from "../../components/Collection/FilterCollection";
-import { GridCollection } from "../../components/Collection/GridCollection";
-import { ListCollection } from "../../components/Collection/ListCollection";
+import { SubHeader } from "../../components/Collection/SubHeader";
 
 import "./scss/index.scss";
 
 const Page = props => {
   const { data, history } = props;
+  const [view, changeView] = useState("grid");
   const handleBack = () => {
     history.push("/");
   };
   const regex = /\s|_|(?=[A-Z])/;
```

Comment:
Delete if not used.

Reply:
It's getting used

Reply:
I think I meant this line: `const regex = /\s|_|(?=[A-Z])/; `